fixture/empty_changes.json

```diff
@@ -0,0 +1,5 @@
+{
+    "last_seq": "5-g1AAAAIreJyVkEsKwjAURZ...",

```

Comment:
Let's use an example revision without the `...` like `10-d9e5b0147af143e5b6d1979378ad957b`